pkg/descheduler/evictions/evictions.go

```diff
@@ -290,6 +290,10 @@ func (ev *evictable) IsEvictable(pod *v1.Pod) bool {
 		checkErrs = append(checkErrs, fmt.Errorf("pod is a static pod"))
 	}
 
+	if !utils.IsPodActive(pod) {
```

Comment:
maybe to be more consistent with the other checks, change this function to `IsPodTerminating` and have it return `pod.DeletionTimestamp != nil`

Reply:
Good idea~ Fix done :)